oddbird/Metecho
# Allow readonly users to be Testers.

## Cute animal pic
_Because everyone likes pictures of [animals](https://unsplash.com/t/animals)._

![](https://images.unsplash.com/photo-1567792825717-84dcd666b715?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1350&q=80)

## Link to Trello card (if applicable)
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._

- https://trello.com/c/6898xd2B

## Description
_The commit messages say what you did; this should explain why and/or how._

## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

- Log in as a readonly GH user
  - Assign yourself as a task Tester
  - You should be able to add/remove yourself, remove yourself as an Epic collaborator, and submit a Task PR review
- Log in as a read-write GH user
  - Add/remove a readonly user from a Task tester (not dev) assignment

## Show me
_Provide screenshots/animated gifs/videos if necessary._

<img width="934" alt="Screen Shot 2021-05-12 at 3 10 19 PM" src="https://user-images.githubusercontent.com/552316/118031449-8a9abc80-b334-11eb-8a0a-a9f2213425fc.png">

![Screen Shot 2021-05-12 at 3 10 46 PM](https://user-images.githubusercontent.com/552316/118031522-a0a87d00-b334-11eb-9e52-e722a77fa284.png)

<img width="482" alt="Screen Shot 2021-05-12 at 3 10 55 PM" src="https://user-images.githubusercontent.com/552316/118031453-8bcbe980-b334-11eb-8ffd-caec8c3508de.png">

<img width="478" alt="Screen Shot 2021-05-12 at 3 11 16 PM" src="https://user-images.githubusercontent.com/552316/118031455-8c648000-b334-11eb-8b76-df25fbbbc70e.png">
